pyproject.toml

```diff
@@ -4,13 +4,19 @@ version = "1.0.1"
 description = "Bulk download tool for CSDA orders"
 readme = "README.md"
 requires-python = ">=3.10"
-dependencies = ["click>=8.2.0", "requests>=2.32.3", "tqdm>=4.67.1"]
-
-[project.scripts]
-csdap-bulk-download = "csdap_bulk_download.cli:cli"
+dependencies = ["requests>=2.32.3"]
```

Comment:
Any thoughts on bumping this down a bit? I could see it being too restrictive for other libraries to use the latest, but also the versions before `2.32.2` had a [CVE](https://nvd.nist.gov/vuln/detail/cve-2024-35195) so I'm inclined to be on the side of "update your code"

Reply:
Yeah, I don't know how many downstreams we expect to be using this library anytime soon. I'm inclined to leave it high until someone complains?